src/config/midi.rs

```diff
@@ -0,0 +1,47 @@
+use crate::Result;
+// use serde::{Deserialize, Deserializer};
```

Comment:
I thought it might be easier to at least partially wire up the deserialization of the MIDI slider config but then bailed on that

In general, mimicked module structure/naming and brought in a lot of code verbatim from `pedals`